languages/ruby/exercises/concept/hashes/log_line_test.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
```

Comment:
There are string literals here, but not at risk of being mutated.  So this line could indicate `false` or my preference, that it is not here at all until it is needed.